src/esp/scene/ObjectControls.h

```diff
@@ -42,8 +44,9 @@ class ObjectControls {
   }
 
  protected:
-  MoveFilterFunc moveFilterFunc_ = [](const vec3f& /*start*/,
-                                      const vec3f& end) { return end; };
+  MoveFilterFunc moveFilterFunc_ =
+      [](CORRADE_UNUSED const Magnum::Vector3& start,
```

Comment:
I liked the `/*start*/` and not a macro better, tbh :) Less stuff to read, less work for the compiler.